pyomo/contrib/mindtpy/tests/test_mindtpy.py

```diff
@@ -446,7 +461,7 @@ def test_OA_nonconvex(self):
 
     def test_iteration_limit(self):
         with SolverFactory('mindtpy') as opt:
-            model = ConstraintQualificationExample()
+            model = ConstraintQualificationExample().clone()
```

Comment:
Why do you both construct a new instance and clone it for this test and the ones below? It seems like you could just construct the instance?

Reply:
You are correct. We don't need to clone the model here. However, we need to clone the model in `model_list`, right? Since the instance is built only once.

Reply:
Yes, that sounds correct.